cmd/ddev/cmd/root_test.go

```diff
@@ -260,16 +255,29 @@ func TestPoweroffOnNewVersion(t *testing.T) {
 	assert.NoError(err)
 	_, err = exec.RunHostCommand(DdevBin, "config", "--auto")
 	assert.NoError(err)
-	_, _ = exec.RunHostCommand(DdevBin, "start", "-y")
+	_, err = exec.RunHostCommand(DdevBin, "start", "-y")
 	assert.NoError(err)
+	t.Cleanup(func() {
```

Comment:
@rfay This cleanup rely on original `HOME` / `USERPROFILE`, so it **must be** called after the cleanup from `t.Setenv` below.